Source/Core/BoundingSphere.js

```diff
@@ -1109,6 +1109,10 @@ define([
                 left.radius === right.radius);
     };
 
+    BoundingSphere.distanceToCamera = function(sphere, frameState) {
+        return Math.max(0.0, Cartesian3.distance(sphere.center, frameState.camera.positionWC) - sphere.radius);
+    };
+
```

Comment:
Throw `DeveloperError` if `sphere` or `frameState` is not defined, similar to what the other functions do.